zilliqa/src/block_store.rs

```diff
@@ -299,4 +299,8 @@ impl BlockStore {
             .entry(peer)
             .or_insert_with(|| PeerInfo::new(capacity))
     }
+
+    pub fn get_num_peers(&self) -> usize {
```

Comment:
Small nit but I think this is now unused?

Reply:
Fixed